test/test_reporters/minitest_reporter_test.rb

```diff
@@ -167,8 +167,76 @@ def test_minitest_spec_output
       )
     end
 
+    def test_prerecord_with_activesupport_prerecord_result_class
+      fake_path = "test/fake_file.rb"
+      uri = URI::Generic.from_path(path: File.expand_path(fake_path)).to_s
+
+      # ActiveSupport parallel testing passes PrerecordResultClass instead of test class
+      prerecord_result = Struct.new(:klass, :source_location).new(
+        "MyTestClass",
+        [fake_path, 42],
+      )
+
+      events = capture_lsp_output do
+        MinitestReporter.new.prerecord(prerecord_result, "test_something")
+      end
+
+      expected = [
+        {
+          "method" => "start",
+          "params" => {
+            "id" => "MyTestClass#test_something",
+            "uri" => uri,
+            "line" => 41, # 42 - 1 for zero-based line numbers
```

Comment:
Almost 100% of our line information in the LSP is always zero based since that's what the editor expects. I don't think we need the comment here.
```suggestion
            "line" => 41,
```